hole/morse.go

```diff
@@ -56,18 +56,28 @@ func morse(reverse bool) []Scorecard {
 	})
 
 	args := []string{strings.Join(words, " ")}
-	out := ""
 
-	for _, char := range args[0] {
-		out += morseMap[char] + "   "
+	if reverse {
```

Comment:
I would remove the if statement, because the Morse Encoder also needs this. Otherwise, this seems to work fine.